src/main/java/org/folio/innreach/domain/service/impl/PatronInfoServiceImpl.java

```diff
@@ -126,6 +127,10 @@ private User findPatronUser(String visiblePatronId, String patronName, VisiblePa
     return user;
   }
 
+  private boolean fieldConfigurationIsNullOrEmpty(VisiblePatronFieldConfiguration fieldConfig){
+    return fieldConfig == null || (fieldConfig.getFields().isEmpty() && fieldConfig.getUserCustomFields().isEmpty());
```

Comment:
wouldn't it be better to completely remove an entity with empty configs on update?